packages/e2e-test-utils/src/index.js

```diff
@@ -77,5 +77,6 @@ export { visitAdminPage } from './visit-admin-page';
 export { waitForWindowDimensions } from './wait-for-window-dimensions';
 export { showBlockToolbar } from './show-block-toolbar';
 export { openPreviewPage } from './preview';
+export { wpDataSelect } from './wp-data-select';
```

Comment:
Is there any reason this function hasn't been exported yet? 🤔

Reply:
> I'd say because this is the first time it's being used outside of e2e-test-utils. 🤔

Most likely.  I wonder if we should be moving our experimental test utils to e2e-test-utils package as opposed to in e2e-tests.  Not a blocker for this PR though.